test/rest-adapter.test.js

```diff
@@ -17,6 +17,10 @@ var factory = require('./helpers/shared-objects-factory.js');
 describe('RestAdapter', function() {
   var remotes;
 
+  before(function() {
+    process.env.NO_DEPRECATION = 'strong-remoting';
+  });
+
```

Comment:
Oh, I think I know what's the problem: `depd` initializes when the file is `require`-d, thus changing `env.NO_DEPRECATION` at runtime has no effect. Can you try to use `process.on('deprecation')` instead? See [docs](https://www.npmjs.com/package/depd#processondeprecation-fn).